### ISSUE TITLE
[Feature Request] Alternative Ways to Raise Speed

### ISSUE THREAD

**Issue by Foxwells** [Kitteh6660/Corruption-of-Champions-Mod#420]
As it stands, the only ways to raise speed are via stat level-up or exploring the lake (which stops after you hit 50 speed). Jojo meditation might as well, I don't recall (it's been a long while since I've gotten stat boosts from it). For people such as myself who've hit max level and don't have any stat points left, it leaves me with incomplete speed when it get lowered, with no way of raising it back up. I would like to see alternative ways to raise speed. Strength and toughness have Vitality Tinctures, and intelligence has spell books and Scholar's Tea. Obviously it wouldn't make sense for there to be a tea for speed, but maybe jogging at the gym could improve it? Or remove the 50 cap at lake exploration-- after all, it isn't like intelligence has any cap, so the cap creates a weird inconsistency.

**Comment by Stadler76** [comment]
Submitting to Akbal is another option to raise your speed to the maximum. And if Jojo is a bimbo AKA Joy, you can train with her to max out your speed, too.

**Comment by Foxwells** [comment]
Bimbo Jojo is a poor choice since there's no way to de-Bimbo him, submitting to Akbal enough times will have an added bad end also making it a poor choice (it works for now, since the bad end doesn't exist, but when it does it'll be a poor choice and I'm suggesting something that wouldn't be affected by something like a bad end being added).

**Comment by Stadler76** [comment]
I don't know of any Akbal related bad end or any plans to implement one. Could you point me to that suggestion?

**Comment by Foxwells** [comment]
http://coc-revamp-mod.wikia.com/wiki/Akbal#Bad_Ends ¯\_(ツ)_/¯ I admit that there's no way of saying when that was added or if it'll still happen, but it's there, so I've no reason to assume it won't be added.

**Comment by Killette2** [comment]
Jogging at the gym does improve speed doesn't it? It's how I raised my speed up to 90 I think

**Comment by Foxwells** [comment]
It just changes body stats for me?

**Comment by Stadler76** [comment]
```as3
private function goJogging():void {
	clearOutput();
	// [snipp]
	//Stat changes HERE!
	if (player.spe < 40) dynStats("spe", .3);
	if (player.tou < 90) dynStats("tou", .5);
	// [snipp]
}
```

In other words, it only raises you speed up to 40. And actually quite slow ...

**Comment by Foxwells** [comment]
Ah. Yeah, my speed is far above 40 by the time I get to Tel'Adre, lol.

**Comment by Killette2** [comment]
It's been so long, guess I can't tell you how I raise my speed then

**Comment by Killette2** [comment]
Succubi milk seems to help even past 90

**Comment by Foxwells** [comment]
Since when has Succubi Milk raised speed? Is it only unpurified that does that? If so, unfair. People should not have to modify themselves just because there is no good way to raise speed.

**Comment by Killette2** [comment]
What about the prison. You can get unlimited speed cardio training there

**Comment by Stadler76** [comment]
Foxwells commented
> Since when has Succubi Milk raised speed? Is it only unpurified that does that? If so, unfair. People should not have to modify themselves just because there is no good way to raise speed.

Both tainted and pure Succubus Milk seems to raise speed by 1 per use (no randomness):
```as3
if (tainted) dynStats("spe", 1, "lus", 3, "cor", 1);
else dynStats("spe", 1, "lus", 3);
```
But I don't wanna use a transformative to raise my speed to the maximum.

**Comment by Foxwells** [comment]
@Killette2 The prison is incomplete, very buggy, game-breaking, and kind of... really unfun past like the first run. I don't want to have to throw myself into that just to max speed. And do we have confirmation it actually has no caps?

@Stadler76 Ah. See, I really only use Incubi Drafts, Salamander Firewater, Drake's Hearts, Eggs, and maybe the occasional Succubus' Delight/Dream, so I wouldn't keep track of anything like that... I suppose the transformative properties can be countered via Hummanus and Reducto, but that's either wasting money or a whole lotta waiting. Not to mention how hard it is to even collect Succubi Milk in the first place... Maybe I just have bad luck, but whenever I'm trying to get Succubi Milk or Incubi Drafts, it takes me days to collect a sufficient amount.

**Comment by Stadler76** [comment]
mmh, how bout building a road around the camp if the wall including the gate is completed? It could be somewhat be connected to other roads and would use stones.
The PC could then do sprints and lap running there.
Later the road could occasionally lure traveling merchants to it, like Giacomo for example.

Or: Improve the gym, so normal lap running has no more cap for your toughness and sprinting would cause significantly more fatigue, but in addition won't have a limit to speed. And maybe for both more than, just the usual +0,3 toughness and speed each time.

**Comment by CapriciousClowns** [comment]
oh the prison workout? yes there is no caps to how much speed you can gain though it is a really slow process

**Comment by Foxwells** [comment]
@CapriciousClowns Just said why that's not really a good idea.

> The prison is incomplete, very buggy, game-breaking, and kind of... really unfun past like the first run. I don't want to have to throw myself into that just to max speed.

@Stadler76 Is there a way to control when you sprint or run a normal lap, though?

**Comment by Stadler76** [comment]
Nope, not yet. Only the normal run laps option. Hence, why I'm suggesting that ^^

**Comment by Xel101stars** [comment]
and it sucks thres no tonixs to raise speed like there is int and str

**Comment by MCmadam** [comment]
if (tainted) dynStats("spe", 1, "lus", 3, "cor", 1);
else dynStats("spe", 1, "lus", 3);

**Comment by MCmadam** [comment]
> As it stands, the only ways to raise speed are via stat level-up, running at Whitney's farm (which is a random event in a pool of others, and takes 2-3 runs to actually go up a point), or exploring the lake (which stops after you hit 50 speed). Jojo meditation might as well, I don't recall (it's been a long while since I've gotten stat boosts from it). For people such as myself who've hit max level and don't have any stat points left, it leaves me with incomplete speed when it get lowered, with no way of raising it back up. I would like to see alternative ways to raise speed. Strength and toughness have Vitality Tinctures, and intelligence has spell books and Scholar's Tea. Obviously it wouldn't make sense for there to be a tea for speed, but maybe jogging at the gym could improve it? Or remove the 50 cap at lake exploration-- after all, it isn't like intelligence has any cap, so the cap creates a weird inconsistency.
